src/components/base/JRaisedButton/JRaisedButton.js

```diff
@@ -11,55 +11,56 @@ import jRaisedButtonStyle from './jRaisedButton.m.scss'
 type Themes = 'blue' | 'white' | 'gray'
 
 type Props = {
-  children: React$Node,
-  theme: Themes,
-  onClick?: Function,
-  isLoading?: boolean,
-  disabled?: boolean,
-  className?: ?string,
+  +onClick: Function,
+  +children: React$Node,
+  +theme: Themes,
+  +className: ?string,
+  +isLoading: boolean,
+  +isDisabled: boolean,
 }
 
 class JRaisedButton extends PureComponent<Props, *> {
   static defaultProps = {
+    onClick: null,
     children: null,
     theme: 'blue',
-    onClick: undefined,
     isLoading: false,
-    disabled: undefined,
+    isDisabled: false,
```

Comment:
I still feel uncomfortable by reason props naming convention for base components. I think, lower case is more aesthetic in this case, case it make a relation with HTML attributes.